### PR TITLE
distrobox: add page

### PR DESCRIPTION
**Pull request by misobarisic**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

**Version of the command being documented (if known): 1.2.10**

### CODE REVIEW

File: pages/linux/distrobox-enter.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# distrobox-enter
+
+> Manual page viewer for distrobox.
```
Comment (navarroaxel) [inline pages/linux/distrobox-enter.md:3]:
```suggestion
> Manual page viewer for distrobox.
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-create.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# distrobox-create
+
+> Manual page viewer for distrobox.
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:3]:
```suggestion
> Manual page viewer for distrobox.
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-rm.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# distrobox-rm
+
+> Manual page viewer for distrobox.
```
Comment (navarroaxel) [inline pages/linux/distrobox-rm.md:3]:
```suggestion
> Manual page viewer for distrobox.
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-rm.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+- Remove a distrobox:
+
+`distrobox-rm container-name`
```
Comment (navarroaxel) [inline pages/linux/distrobox-rm.md:7]: We use `snake_case` for placeholders
```suggestion
`distrobox-rm {{container_name}}`
```

File: pages/linux/distrobox-rm.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+- Remove a distrobox forcefully:
+
+`distrobox-rm container-name --force`
```
Comment (navarroaxel) [inline pages/linux/distrobox-rm.md:11]:
```suggestion
`distrobox-rm {{container_name}} --force`
```

File: pages/linux/distrobox-list.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# distrobox-list
+
+> Manual page viewer for distrobox.
+
+- List all distrobox containers:
+
+`distrobox-list`
+
+- List all distrobox containers (verbose):
```
Comment (navarroaxel) [inline pages/linux/distrobox-list.md:9]:
```suggestion
- List all distrobox containers with verbose information:
```

File: pages/linux/distrobox-create.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+- Create a distrobox:
+
+`distrobox-create my-alpine-container --image alpine`
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:7]:
```suggestion
`distrobox-create {{container_name}} --image {{alpine}}`
```

File: pages/linux/distrobox-create.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+- Clone a distrobox:
+
+`distrobox-create --clone ubuntu-container ubuntu-copy`
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:11]:
```suggestion
`distrobox-create --clone {{container_name}} {{cloned_container_name}}`
```

File: pages/linux/distrobox-create.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# distrobox-create
+
+> Manual page viewer for distrobox.
+
+- Create a distrobox:
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:5]:
```suggestion
- Create a distrobox using the Alpine image:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by navarroaxel** [comment]
Hi @misobarisic, you marked my suggestions as resolved, but I can't see the changes applied.

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/distrobox-create.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# distrobox-create
+
+> Manual page viewer for distrobox.
+> More information: <https://distrobox.privatedns.org>
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:4]:
```suggestion
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-enter.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://distrobox.privatedns.org>
```
Comment (navarroaxel) [inline pages/linux/distrobox-enter.md:4]:
```suggestion
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-list.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
+# distrobox-list
+
+> Manual page viewer for distrobox.
+> More information: <https://distrobox.privatedns.org>
```
Comment (navarroaxel) [inline pages/linux/distrobox-list.md:4]:
```suggestion
> More information: <https://distrobox.privatedns.org>.
```

File: pages/linux/distrobox-rm.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
+# distrobox-rm
+
+> Manual page viewer for distrobox.
+> More information: <https://distrobox.privatedns.org>
```
Comment (navarroaxel) [inline pages/linux/distrobox-rm.md:4]:
```suggestion
> More information: <https://distrobox.privatedns.org>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/distrobox-create.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+# distrobox-create
+
+> Manual page viewer for distrobox.
```
Comment (sbrl) [inline pages/linux/distrobox-create.md:3]: All these descriptions for these pages are the same. The description should describe what the command does.

**Comment by misobarisic** [comment]
All pages have been updated with their corresponding descriptions.

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/distrobox-create.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # distrobox-create
 
-> distrobox-create takes care of creating the container with input name and image.
+> This command takes care of creating the container with input name and image.
```
Comment (navarroaxel) [inline pages/linux/distrobox-create.md:3]:
```suggestion
> Create Distrobox containers with input name and image.
```

File: pages/linux/distrobox-enter.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 # distrobox-enter
 
-> distrobox-enter takes care of entering the container with the name specified.
+> This command takes care of entering the container with the name specified.
```
Comment (navarroaxel) [inline pages/linux/distrobox-enter.md:3]: Perhaps this is more clear.
```suggestion
> Run a command in a Distrobox container.
```

File: pages/linux/distrobox-rm.md [hunk 17]
```diff
@@ -1,6 +1,6 @@
 # distrobox-rm
 
-> distrobox-rm delete one of the available distroboxes.
+> This command deletes one of the available distroboxes.
```
Comment (navarroaxel) [inline pages/linux/distrobox-rm.md:3]:
```suggestion
> Delete Distrobox containers.
```

File: pages/linux/distrobox-list.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # distrobox-list
 
-> distrobox-list lists available distroboxes. It detects them and lists them separately from the rest of normal podman or docker containers.
+> This command lists available distroboxes. It detects them and lists them separately from the rest of normal podman or docker containers.
```
Comment (navarroaxel) [inline pages/linux/distrobox-list.md:3]:
```suggestion
> List available Distrobox containers. It detects them and lists them separately from the rest of normal podman or Docker containers.
```

File: pages/linux/distrobox-create.md (hunk 15 above)
Comment (CleanMachine1) [reply pages/linux/distrobox-create.md:3]:
```suggestion
> Create Distrobox containers with an input name and image.
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**